app/views/ChooseProvider.js

```diff
@@ -19,27 +17,22 @@ import {
   renderers,
   withMenuContext,
 } from 'react-native-popup-menu';
-import RNFetchBlob from 'rn-fetch-blob';
 
-import backArrow from './../assets/images/backArrow.png';
 import closeIcon from './../assets/images/closeIcon.png';
 import saveIcon from './../assets/images/saveIcon.png';
 import { DynamicTextInput } from '../components/DynamicTextInput';
 import NavigationBarWrapper from '../components/NavigationBarWrapper';
 import { Typography } from '../components/Typography';
-import { AUTHORITIES_LIST_URL } from '../constants/authorities';
-import colors from '../constants/colors';
 import Colors from '../constants/colors';
 import fontFamily from '../constants/fonts';
 import { AUTHORITY_SOURCE_SETTINGS } from '../constants/storage';
-import { GetStoreData, SetStoreData } from '../helpers/General';
+import { SetStoreData } from '../helpers/General';
 import { checkIntersect } from '../helpers/Intersect';
 import languages from '../locales/languages';
+import HCAService from '../services/HCAService';
```

Comment:
prefer named exports because they work better with VSCode and other IDEs (they're more discoverable)

Reply:
Done.